server/models/ParticipantsModel/index.js

```diff
@@ -148,9 +120,9 @@ const allParticipantsQuery = (user, queryParams) => {
               },
             },
             {
-              participant: {
-                $in: searchParticipants,
-              },
+              $expr: {
```

Comment:
Document DB is missing support $expr 😢 

<img width="600" alt="Screenshot 2024-03-12 at 12 18 03 PM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/553359f2-be21-4531-8332-df7fd8c78e1e">

Reply:
Gotcha, will look into a fix

Reply:
[18a4ef0](https://github.com/AMP-SCZ/dpdash/pull/676/commits/18a4ef0a8645d1c31d9e28a3eec8118b0fb3a720)